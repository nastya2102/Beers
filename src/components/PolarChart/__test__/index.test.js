import React from 'react'
import { render } from '@testing-library/react'
import PolarChart from '../index'

describe('PolarChart', () => {
  test('should render PolarChart component', () => {
    const { container } = render(<PolarChart data={[1, 2]} allData={[{}, {}]} handleClickBeer={() => {}}/>)

    expect(container).toBeTruthy()
  })

  test('should match snapshot PolarChart component', () => {
    const { container } = render(<PolarChart data={[1, 2]} allData={[{}, {}]} handleClickBeer={() => {}}/>)

    expect(container).toMatchSnapshot()
  })
})
