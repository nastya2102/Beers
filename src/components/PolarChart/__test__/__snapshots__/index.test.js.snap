// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PolarChart should match snapshot PolarChart component 1`] = `
<div>
  <canvas
    height="150"
    width="300"
  />
</div>
`;
